chains/manager.go

```diff
@@ -1351,6 +1356,7 @@ func (m *manager) createSnowmanChain(
 
 	// create bootstrap gear
 	bootstrapCfg := smbootstrap.Config{
+		Appraiser:                      proposerVM,
```

Comment:
If we choose to pass this in through a separate variable - it seems like we could re-use the `block.Parser` interface and just wrap the proposervm in an unexported struct to re-direct the `ParseBlock` -> `AppraiseBlock`

Reply:
Changed accordingly. I'm a bit uneasy about this though, as now if someone uses the parser for anything that does require a signature verification, it will be silently locally verified. 

When you have an explicit method call (e.g LocalParseBlock) you as a caller carry the responsibility of the semantics.